rustls/src/msgs/message.rs

```diff
@@ -248,6 +245,16 @@ impl<'a> BorrowedOpaqueMessage<'a> {
     }
 }
 
+#[derive(Debug)]
+pub enum MessageError {
```

Comment:
Please make this move in a separate commit.

Reply:
May I suggest it is done in a broader reorder of the file (as you suggested [here](https://github.com/rustls/rustls/pull/1640#discussion_r1484045051)), not in this PR?

Reply:
That's fine too.